src/components/Invitations/InviteProvider.tsx

```diff
@@ -35,6 +35,32 @@ export type InviteProviderProps = {
   children: React.ReactNode;
 };
 
+const isInviteAlreadyAcceptedErrorResponse =
+  /* istanbul ignore next */
+  (response: unknown) => {
+    if (!response) {
+      return false;
+    }
+
+    if (typeof response !== 'object') {
+      return;
+    }
+
+    if ('code' in response && response.code === 'INVITATION_ALREADY_ACCEPTED') {
+      return true;
+    }
+
+    if (
+      'error' in response &&
+      typeof response.error === 'string' &&
+      response.error.includes('already accepted')
```

Comment:
Should the API be modified to have a consistent behavior instead of two ways to return the same thing?

Reply:
@mdlavin Yes, it definitely should. I'd like to consider that change out-of-scope for this PR.

FWIW, as far as I can tell, the behavior is consistent. But, the current SDK logic is simply not correct. I'd like to get this sub-optimal extra handling in place, then follow-up with a deeper investigation to simplify the logic (client-side, server-side, or both)